### ISSUE TITLE
Support for "fixed" RedisLabs subscriptions

### ISSUE THREAD

**Issue by johndorian93** [RedisLabs/terraform-provider-rediscloud#118]
Hello.

It is more a question than an issue to fix. RedisLabs provides two types of subscriptions: 
- "fixed" - for low-throughput use cases
- "flexible" - for any dataset size or throughput

I want to create a "fixed" subscription using your terraform-provider-rediscloud. However, it creates a "flexible" subscription by default. Is there any way to force the provider to create the "fixed" subscription?

Thanks in advance.

**Comment by alinatok** [comment]
Hi,

I'm facing the same issue. @johndorian93 did you find a workaround?

Pricing difference between Fixed and Flexible is huge so at lease for development environment I would like to have a Fixed subscription.

**Comment by alinatok** [comment]
Hi,

I've got a reply from Redislab stating the following:

> Flexible and Fixed subscriptions are conceptually different:
Fixed - Runs on a shared environment, intended for low usage and non-critical data loads. It can only scale in memory and up to a specific limit, not meant for performance. It does not support clustering, OSS API, and other options that might be needed for an enterprise-grade system.
Flexible - Runs on a dedicated environment, intended for production and enterprise-grade. Can scale both in memory and performance almost unlimited.
>
> Due to this nature, Terraform and API support are available for Flexible subscriptions only.
